### DISCUSSION TITLE
Linux iNAV

### DISCUSSION THREAD

**Discussion by RJD2two** [iNavFlight/inav#7586] [General]
Hi, I couldnt get iNAV to work on my linux distro as pretty new to it. Would it be possible of updating iNAV for linux so it is as straight forward as betaflight and expresslrs to install and run? Whether that is .deb, flatpak, snap packages or something else as simple? One click install would be great. 

Apologies if this has been asked before, I did perform a search but did not come across the same question.

Thanks

**Comment by stronnag** [comment]
Don't the instructions [here](http://seyrsnys.myzen.co.uk/inav-configurator-next/#usage) work for you. If not, please say what's missing / unclear and they can be updated.

**Reply by OptimusTi** [reply]
Is that document listed on the official Inav documention?

**Reply by RJD2two** [reply]
Thanks for the quick response. I will follow these instructions provided, I had followed different previously. 

The installation is still more complicated than for example betaflight, especially for someone new to linux. Using ubuntu based distro its as simple as opening the .deb file. Were iNAV a snap package, would it not make installation much more straight forward for clueless linux users like myself over all distros?

**Reply by RJD2two** [reply]
> Is that document listed on the official Inav documention?

No that is not what is on the iNAV page. https://github.com/iNavFlight/inav-configurator

I found this more helpful https://askubuntu.com/questions/1267757/install-inav-configurator-in-ubuntu-mate
